addons/logistics_wirecutter/XEH_postInit.sqf

```diff
@@ -2,6 +2,7 @@
 
 if (hasInterface) then {
     ["ace_interactMenuOpened", {_this call FUNC(interactEH)}] call CBA_fnc_addEventHandler;
+    GVAR(possibleWirecutters) = configProperties [configFile >> "CfgWeapons", "getNumber (_x >> "ACE_isWirecutter") == 1", true];
```

Comment:
Don't the double quotes around `ACE_isWirecutter` create an invalid syntax since the condition string starts with a double quote ?
```suggestion
    GVAR(possibleWirecutters) = configProperties [configFile >> "CfgWeapons", "getNumber (_x >> 'ACE_isWirecutter') == 1", true];
```

Reply:
Thanks, had been wondering why it had done that, my bad